receiver/dummyreceiver/metrics_receiver.go

```diff
@@ -13,15 +13,15 @@ import (
 	"go.uber.org/zap"
 )
 
-type dummyMetricsReceiver struct {
+type dummyreceiver struct {
 	config       *Config
 	nextConsumer consumer.Metrics
 	settings     *receiver.Settings
 
 	cancel context.CancelFunc
 }
 
-func (r *dummyMetricsReceiver) Start(_ context.Context, _ component.Host) error {
+func (r *dummyreceiver) Start(_ context.Context, _ component.Host) error {
```

Comment:
Can you please write a log message saying staring dummy receiver

Reply:
ahh, missed that